server/ethereum/ethereum.test.js

```diff
@@ -63,9 +63,9 @@ beforeAll(async done => {
   done()
 })
 
-beforeEach(async done => {
+beforeEach(() => {
   smartContracts.clear()
-  done()
+  expect(smartContracts.get().length).toBe(0)
```

Comment:
good to double-check

Reply:
Agreed, but `smartContracts.get` is going to be gone soon.